### DISCUSSION TITLE
Workflow options while trying to load in (Type:select) shows in html format after opening object

### DISCUSSION THREAD

**Discussion by manojthotla25** [pimcore/pimcore#10880] [Q&A / Support] [answered]
Hi Team,

Task: created a Workflow and trying to fetch them in attribute status which is of type select.

Issue: It shows in Html format. Please see the attached.

![image](https://user-images.githubusercontent.com/93549976/142875259-53d6e13b-404b-4b92-bd14-660a15ec8ff2.png)

When we click on drop down it shows properly. Please see attached.

![image](https://user-images.githubusercontent.com/93549976/142875735-953633e0-02f4-4846-9ce2-089ce32e5298.png)

Can you please tell me How to show them in proper way like all dropdowns shows.

workflows.yml:

pimcore:
    workflows:

        simple_product_workflow:

            label: 'Product Workflow'

            type: 'state_machine'
            marking_store:
                type: single_state
                arguments:
                    - status

            support_strategy:
                type: "expression"
                arguments:
                    - \Pimcore\Model\dataObject\Product
                    - 'is_fully_authenticated()'
            
    

            places:
                start:
                    label: 'Product Launch'
                    title: 'Product Launch'
                    color: '#3572b0'
                    visibleInHeader: true
                    

                input_attributes:
                    label: 'Under Validation'
                    color: 'orange'
                    visibleInHeader: true

                review_attributes:
                    label: 'Validation Sucessful'
                    color: 'green'
                    visibleInHeader: true
                    

            transitions:
                start_workflow:
                    from: start
                    to: input_attributes
                    options:
                        label: 'Start Workflow'
              
                        notificationSettings:
                         - notifyUsers: ['admin']
                           channelType: ['mail', 'pimcore_notification']
                           mailPath: '@PimcoreCore/Workflow/NotificationEmail/notificationEmail.html.twig'
                              
                finish_review:
                    from: input_attributes
                    to: review_attributes
                    options:
                        label: 'Reviewed Sucessful'

     
![image](https://user-images.githubusercontent.com/93549976/142876065-0754a6fb-77c9-483b-8405-24eba618d33c.png)

**Comment by jdreesen** [comment, marked as answer]
Seems like this PR could fix it: #10871
